release/ray_release/test_automation/ci_state_machine.py

```diff
@@ -0,0 +1,42 @@
+from ray_release.test_automation.state_machine import TestStateMachine
+
+from ray_release.test import Test, TestState
+
+
+class CITestStateMachine(TestStateMachine):
```

Comment:
how is this different from the other test state machine?

Reply:
CI state machine: https://docs.google.com/document/d/1IVgI7LNo21bPJO8kdYTwdWwEdZr5D9VtNrabUeX8gCc/edit

Release state machine: https://docs.google.com/document/d/1hV8avu2Xl16Di3zVXyFtjuhZ8OGFDWhEt7Am319bki0/edit#heading=h.tdxb82dxifm7

I think the main difference is that CI tests have flaky state, which can be computed through automation; while release tests since they run so less frequent, the flaky state is managed by human (the unstable tag)